### PR TITLE
🐛Change ampproject.org references to amp.dev

### LINKED ISSUE #21965: Update ampproject.org references in amphtml repo to amp.dev
There are a large number of references to ampproject.org in the amphtml repo that should be updated to amp.dev.

https://github.com/ampproject/amphtml/search?q=%22ampproject.org%22+NOT+%22cdn.ampproject.org%22&unscoped_q=%22ampproject.org%22+NOT+%22cdn.ampproject.org%22&type=Code

/cc @ampproject/wg-outreach

### PR DESCRIPTION
**Pull request by keevkeev**
- References to the old site amproject.org should reference their amp.dev equivalent.
  - Fixes #21965

### CODE REVIEW

File: build-system/SERVING.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ gulp build --css-only
 gulp dist --version 123456789 --type prod --hostname cdn.myowncdn.org --hostname3p 3p.myowncdn.net
 mkdir -p /path/to/cdn/production/
 mkdir -p /path/to/cdn/3p/
-# this would be the files hosted on www.ampproject.org/
+# this would be the files hosted on www.amp.dev/
```
Comment (sebastianbenz) [inline build-system/SERVING.md]: s/www.amp.dev/amp.dev

File: builtins/amp-layout.md [hunk 2]
```diff
@@ -19,17 +19,17 @@ limitations under the License.
...
+    <td class="col-fourty"><strong><a href="https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/control_layout">Supported Layouts</a></strong></td>
     <td>container, fill, fixed, fixed-height, flex-item, intrinsic, responsive</td>
   </tr>
 </table>
 
 ## Overview
 
-The `amp-layout` component allows you to apply aspect-ratio based responsive layouts to any element. The `amp-layout` component works similarly to the [layout](https://www.ampproject.org/docs/guides/responsive/control_layout#the-layout-attribute) attribute on existing AMP components, but supports any HTML markup as children. Other supported layouts all work with `amp-layout` (e.g., fixed-height, fixed, etc.).
+The `amp-layout` component allows you to apply aspect-ratio based responsive layouts to any element. The `amp-layout` component works similarly to the [layout](https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/control_layout?#the-layout-attribute) attribute on existing AMP components, but supports any HTML markup as children. Other supported layouts all work with `amp-layout` (e.g., fixed-height, fixed, etc.).
```
Comment (sebastianbenz) [inline builtins/amp-layout.md]: query param `?#` looks wrong here: `entation/guides-and-tutorials/develop/style_and_layout/control_layout?#the-layout-attribute`

File: extensions/amp-form/amp-form.md [hunk 3]
```diff
@@ -133,7 +133,7 @@ The `amp-form` element exposes the following actions:
 | `clear` | Empties the values from each input in the form. This can allow users to quickly fill out forms a second time. |
 
 {% call callout('Read on', type='read') %}
-Learn more about [Actions and Events in AMP](https://www.ampproject.org/docs/interaction_dynamic/amp-actions-and-events.html).
+Learn more about [Actions and Events in AMP](https://amp.dev/documentation/guides-and-tutorials/learn/amp-actions-and-events.html).
```
Comment (sebastianbenz) [inline extensions/amp-form/amp-form.md]: Use the canonical URL https://amp.dev/documentation/guides-and-tutorials/learn/amp-actions-and-events

File: extensions/amp-iframe/0.1/amp-iframe.js [hunk 4]
```diff
@@ -156,7 +156,7 @@ export class AmpIframe extends AMP.BaseElement {
         'amphtml/blob/master/spec/amp-iframe-origin-policy.md for details.',
         element);
     userAssert(!(endsWith(hostname, `.${urls.thirdPartyFrameHost}`) ||
-        endsWith(hostname, '.ampproject.org')),
+        endsWith(hostname, '.amp.dev')),
```
Comment (sebastianbenz) [inline extensions/amp-iframe/0.1/amp-iframe.js]: I don't think that his ampproject.org should be replaced here as this has nothing to do with the documentation.

File: extensions/amp-script/amp-script.md [hunk 5]
```diff
@@ -49,7 +49,7 @@ Allows rendering of custom UI components running on third-party JavaScript.
 The `amp-script` component allows you to render widgets and other UI using custom third-party JavaScript, e.g. a React component.
 
 {% call callout('Important', type='caution') %}
-`amp-script` is in active development and under [experimental availability](https://www.ampproject.org/docs/reference/experimental.html). It's subject to breaking API changes and should not yet be used in production.
+`amp-script` is in active development and under [experimental availability](https://amp.dev/documentation/guides-and-tutorials/learn/experimental.html). It's subject to breaking API changes and should not yet be used in production.
```
Comment (sebastianbenz) [inline extensions/amp-script/amp-script.md]: Use the canonical url https://amp.dev/documentation/guides-and-tutorials/learn/experimental

File: extensions/amp-story/amp-story.md [hunk 6]
```diff
@@ -31,21 +31,21 @@ limitations under the License.
   </tr>
   <tr>
     <td width="40%"><strong>Availability</strong></td>
-    <td><div><a href="https://www.ampproject.org/docs/reference/experimental.html">Experimental</a></td>
+    <td><div><a href="https://amp.dev/documentation/guides-and-tutorials/learn/experimental.html">Experimental</a></td>
```
Comment (sebastianbenz) [inline extensions/amp-story/amp-story.md]: Use the canonical url https://amp.dev/documentation/guides-and-tutorials/learn/experimental

File: extensions/amp-story/amp-story.md [hunk 7]
```diff
@@ -1432,6 +1432,6 @@ Additionally, for right-to-left languages, you may include the `dir="rtl"` attri
 
 ## Related resources
 
-* [Tutorial: Create a visual AMP story](https://www.ampproject.org/docs/tutorials/visual_story)
+* [Tutorial: Create a visual AMP story](https://amp.dev/documentation/guides-and-tutorials/start/visual_story/)
 * [Samples on AMP By Example](https://ampbyexample.com/stories/#stories/introduction)
```
Comment (sebastianbenz) [inline extensions/amp-story/amp-story.md]: Can you also change the ampbyexample link

File: extensions/amp-viz-vega/amp-viz-vega.md [hunk 8]
```diff
@@ -28,14 +28,14 @@ Displays visualizations created using <a href="https://vega.github.io/vega/">Veg
 <table>
   <tr>
     <td width="40%"><strong>Availability</strong></td>
-    <td><a href="https://www.ampproject.org/docs/reference/experimental.html">Experimental</a></td>
+    <td><a href="https://amp.dev/documentation/guides-and-tutorials/learn/experimental.html">Experimental</a></td>
```
Comment (sebastianbenz) [inline extensions/amp-viz-vega/amp-viz-vega.md]: Use the canonical URL https://amp.dev/documentation/guides-and-tutorials/learn/experimental

File: spec/amp-actions-and-events.md [hunk 9]
```diff
@@ -633,15 +633,15 @@ actions that apply to the whole document.
   <tr>
     <td><code>setState({foo: 'bar'})</code><sup>1</sup></td>
     <td>
-      <p>Requires <a href="https://www.ampproject.org/docs/reference/components/amp-bind.html#updating-state-with-ampsetstate">amp-bind</a>.</p>
+      <p>Requires <a href="https://amp.dev/documentation/components/amp-bind.html#updating-state-with-ampsetstate">amp-bind</a>.</p>
```
Comment (sebastianbenz) [inline spec/amp-actions-and-events.md]: use the canonical URL without `.html`

File: spec/amp-actions-and-events.md [hunk 10]
```diff
@@ -633,15 +633,15 @@ actions that apply to the whole document.
...
       <p>Merges an object literal into the bindable state.</p>
       <p></p>
     </td>
   </tr>
   <tr>
     <td><code>pushState({foo: 'bar'})</code><sup>1</sup></td>
     <td>
-      <p>Requires <a href="https://www.ampproject.org/docs/reference/components/amp-bind.html#modifying-history-with-amppushstate">amp-bind</a>.</p>
+      <p>Requires <a href="https://amp.dev/documentation/components/amp-bind.html#modifying-history-with-amppushstate">amp-bind</a>.</p>
```
Comment (sebastianbenz) [inline spec/amp-actions-and-events.md]: use the canonical URL without `.html`

File: spec/amp-actions-and-events.md [hunk 11]
```diff
@@ -650,11 +650,11 @@ actions that apply to the whole document.
 
 ### Target: amp-access
 
-The `amp-access` target is provided by the [amp-access](https://www.ampproject.org/docs/reference/components/amp-access.html) component.
+The `amp-access` target is provided by the [amp-access](https://amp.dev/documentation/components/amp-access.html) component.
```
Comment (sebastianbenz) [inline spec/amp-actions-and-events.md]: use the canonical URL without `.html`

File: spec/amp-cors-requests.md [hunk 12]
```diff
@@ -403,7 +403,7 @@ To ensure your cached AMP page renders and functions correctly:
 1.  From your browser, open the URL that the AMP Cache would use to access your AMP page. You can determine the cache URL format from this [tool on AMP By Example](https://ampbyexample.com/advanced/using_the_google_amp_cache/).
 
     For example:
-    * URL: `https://www.ampproject.org/docs/tutorials/create.html`
+    * URL: `https://amp.dev/documentation/guides-and-tutorials/start/create/`
     * AMP Cache URL format: `https://www-ampproject-org.cdn.ampproject.org/c/s/www.ampproject.org/docs/tutorials/create.html`
```
Comment (sebastianbenz) [inline spec/amp-cors-requests.md]: This URL also needs to be adjusted

File: tools/experiments/experiments.html [hunk 13]
```diff
@@ -165,7 +165,7 @@ <h1>AMP Experiments</h1>
       <a href="https://developers.google.com/amp/cache/">Google AMP Cache</a>;
       that is, the cached copies of valid AMP content published to the web at
       the cdn.ampproject.org domain. See the AMP project
-      <a href="https://www.ampproject.org/docs/reference/experimental.html">documentation</a>
+      <a href="https://amp.dev/documentation/guides-and-tutorials/learn/experimental.html">documentation</a>
```
Comment (sebastianbenz) [inline tools/experiments/experiments.html]: use the canonical URL without `.html`

File: validator/chromeextension/manifest.json [hunk 14]
```diff
@@ -27,7 +27,7 @@
     }
   ],
   "content_security_policy": "script-src 'self' https://cdn.ampproject.org; object-src 'self'",
-  "homepage_url": "https://www.ampproject.org",
+  "homepage_url": "https://amp.dev/",
```
Comment (sebastianbenz) [inline validator/chromeextension/manifest.json]: remove the trailing slash

File: validator/gulpjs/package.json [hunk 15]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "gulp-amphtml-validator",
   "version": "1.0.2",
-  "description": "Gulp plugin for the official AMP HTML validator (www.ampproject.org)",
+  "description": "Gulp plugin for the official AMP HTML validator (www.amp.dev)",
```
Comment (sebastianbenz) [inline validator/gulpjs/package.json]: s/www.amp.dev/amp.dev

File: validator/nodejs/README.md [hunk 16]
```diff
@@ -41,8 +41,8 @@ Now try running it:
 ```
 $ node demo.js
 FAIL
-line 1, col 0: The mandatory attribute '⚡' is missing in tag 'html ⚡ for top-level html'. (see https://www.ampproject.org/docs/reference/spec.html#required-markup)
-line 1, col 0: The parent tag of tag 'html ⚡ for top-level html' is '$root', but it can only be '!doctype'. (see https://www.ampproject.org/docs/reference/spec.html#required-markup)
+line 1, col 0: The mandatory attribute '⚡' is missing in tag 'html ⚡ for top-level html'. (see https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml.html#required-markup)
```
Comment (sebastianbenz) [inline validator/nodejs/README.md]: remove `.html`

File: validator/nodejs/README.md [hunk 17]
```diff
@@ -41,8 +41,8 @@ Now try running it:
...
+line 1, col 0: The parent tag of tag 'html ⚡ for top-level html' is '$root', but it can only be '!doctype'. (see https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml.html#required-markup)
```
Comment (sebastianbenz) [inline validator/nodejs/README.md]: remove `.html`

File: validator/nodejs/package.json [hunk 18]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "amphtml-validator",
   "version": "1.0.23",
-  "description": "Official validator for AMP HTML (www.ampproject.org)",
+  "description": "Official validator for AMP HTML (www.amp.dev)",
```
Comment (sebastianbenz) [inline validator/nodejs/package.json]: s/www.amp.dev/amp.dev

File: validator/package.json [hunk 19]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "amp-validator",
   "version": "0.1.0",
-  "description": "Validator for AMP HTML (www.ampproject.org)",
+  "description": "Validator for AMP HTML (www.amp.dev)",
```
Comment (sebastianbenz) [inline validator/package.json]: s/www.amp.dev/amp.dev

File: validator/testdata/amp4ads_feature_tests/amp_story_ad.html [hunk 20]
```diff
@@ -25,7 +25,7 @@
     <meta name="viewport" content="width=device-width,minimum-scale=1">
 
     <!-- Specifies where the user is directed -->
-    <meta name="amp-cta-url" content="https://www.ampproject.org">
+    <meta name="amp-cta-url" content="https://amp.dev/">
```
Comment (sebastianbenz) [inline validator/testdata/amp4ads_feature_tests/amp_story_ad.html]: remove the trailing slash

File: validator/testdata/amp4ads_feature_tests/amp_story_ad_errors.html [hunk 21]
```diff
@@ -25,7 +25,7 @@
     <meta name="viewport" content="width=device-width,minimum-scale=1">
 
     <!-- Should fail as an invalid meta tag. -->
-    <meta name="amp-cta" content="https://www.ampproject.org">
+    <meta name="amp-cta" content="https://amp.dev/">
```
Comment (sebastianbenz) [inline validator/testdata/amp4ads_feature_tests/amp_story_ad_errors.html]: remove the trailing slash

File: validator/webui/@polymer/ampproject-toolbar/ampproject-toolbar.html [hunk 22]
```diff
@@ -57,12 +57,12 @@
   <template>
     <paper-toolbar>
       <div class="ampproject-left ampproject-link">
-        <a href="https://www.ampproject.org/"
+        <a href="https://amp.dev/"
```
Comment (sebastianbenz) [inline validator/webui/@polymer/ampproject-toolbar/ampproject-toolbar.html]: remove the trailing slash

**Comment by keevkeev** [comment]
thanks for the review @sebastianbenz. Since there are quite a few references to ampbyexample I will take care of those links in a separate PR

File: validator/webui/@polymer/ampproject-toolbar/ampproject-toolbar.html (hunk 22 above)
Comment (keevkeev) [reply validator/webui/@polymer/ampproject-toolbar/ampproject-toolbar.html]: fixed

File: validator/testdata/amp4ads_feature_tests/amp_story_ad_errors.html (hunk 21 above)
Comment (keevkeev) [reply validator/testdata/amp4ads_feature_tests/amp_story_ad_errors.html]: fixed

File: validator/testdata/amp4ads_feature_tests/amp_story_ad.html (hunk 20 above)
Comment (keevkeev) [reply validator/testdata/amp4ads_feature_tests/amp_story_ad.html]: fixed

File: build-system/SERVING.md (hunk 1 above)
Comment (keevkeev) [reply build-system/SERVING.md]: fixed

File: validator/webui/@polymer/ampproject-toolbar/ampproject-toolbar.html (hunk 22 above)
Comment (keevkeev) [reply validator/webui/@polymer/ampproject-toolbar/ampproject-toolbar.html]: fixed

File: validator/package.json (hunk 19 above)
Comment (keevkeev) [reply validator/package.json]: fixed

File: validator/nodejs/package.json (hunk 18 above)
Comment (keevkeev) [reply validator/nodejs/package.json]: fixed

File: extensions/amp-script/amp-script.md (hunk 5 above)
Comment (keevkeev) [reply extensions/amp-script/amp-script.md]: fixed

File: extensions/amp-iframe/0.1/amp-iframe.js (hunk 4 above)
Comment (keevkeev) [reply extensions/amp-iframe/0.1/amp-iframe.js]: fixed

File: extensions/amp-form/amp-form.md (hunk 3 above)
Comment (keevkeev) [reply extensions/amp-form/amp-form.md]: fixed

File: builtins/amp-layout.md (hunk 2 above)
Comment (keevkeev) [reply builtins/amp-layout.md]: fixed

File: extensions/amp-story/amp-story.md (hunk 6 above)
Comment (keevkeev) [reply extensions/amp-story/amp-story.md]: fixed

File: extensions/amp-viz-vega/amp-viz-vega.md (hunk 8 above)
Comment (keevkeev) [reply extensions/amp-viz-vega/amp-viz-vega.md]: fixed

File: spec/amp-actions-and-events.md (hunk 9 above)
Comment (keevkeev) [reply spec/amp-actions-and-events.md]: fixed

File: spec/amp-actions-and-events.md (hunk 10 above)
Comment (keevkeev) [reply spec/amp-actions-and-events.md]: fixed

**Comment by sebastianbenz** [comment]
@keevkeev can you check why the build is failing?

**Comment by keevkeev** [comment]
@sebastianbenz after the merge the build is working for me again now

File: extensions/amp-access/0.1/amp-access-server-jwt.js [hunk 23]
```diff
@@ -38,7 +38,7 @@ const TAG = 'amp-access-server-jwt';
 const AUTHORIZATION_TIMEOUT = 3000;
 
 /** @const {string} */
-const AMP_AUD = 'ampproject.org';
+const AMP_AUD = 'amp.dev';
```
Comment (sebastianbenz) [inline extensions/amp-access/0.1/amp-access-server-jwt.js]: This should stay at ampproject.org

File: extensions/amp-access/0.1/test/test-amp-access-server-jwt.js [hunk 24]
```diff
@@ -581,7 +581,7 @@ describes.realWin('AccessServerJwtAdapter', {amp: true}, env => {
       beforeEach(() => {
         jwt = {
           'exp': Math.floor((Date.now() + 10000) / 1000),
-          'aud': 'ampproject.org',
+          'aud': 'amp.dev',
```
Comment (sebastianbenz) [inline extensions/amp-access/0.1/test/test-amp-access-server-jwt.js]: this needs to be ampproject.org (for the whole file)

Comment (keevkeev) [reply extensions/amp-access/0.1/test/test-amp-access-server-jwt.js]: fixed

File: extensions/amp-access/0.1/amp-access-server-jwt.js (hunk 23 above)
Comment (keevkeev) [reply extensions/amp-access/0.1/amp-access-server-jwt.js]: fixed

**Comment by honeybadgerdontcare** [comment]
Also please revert any changes to `.protoascii` and `.out` files within `extensions/`.

**Comment by rsimha** [comment]
@keevkeev I'm seeing that https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/1173754/ is a dead link, so I don't think it should be whitelisted. Instead, I suggest changing the link to https://developer.microsoft.com/en-us/microsoft-edge/platform/status/supportcsstransformsonsvg, which deals with the same topic.

**Comment by keevkeev** [comment]
@rsimha thanks for the link, this PR got really out of hand real quick. Thank you for your patience and sorry I messed it up so badly. all build checks have passed now so I think you could go ahead an start reviewing

File: ads/google/a4a/docs/RTBExchangeGuide.md [hunk 25]
```diff
@@ -1,82 +1,82 @@
 # AMPHTML Ad Implementation Guide for RTB Ad Exchanges
 ---
 ## Objective
-
```
Comment (rsimha) [inline ads/google/a4a/docs/RTBExchangeGuide.md]: I believe trailing spaces have a special purpose in `.md` files, and shouldn't automatically be removed.

I assume your editor did this. There's a way to disable the behavior for vscode: https://github.com/Microsoft/vscode/issues/1679

Can you revert these changes?

Comment (keevkeev) [reply ads/google/a4a/docs/RTBExchangeGuide.md]: @rsimha I believe this issue is resolved, can you confirm?

Comment (rsimha) [reply ads/google/a4a/docs/RTBExchangeGuide.md]: It isn't. See https://github.com/ampproject/amphtml/pull/22126/files#diff-12cfb8e62838a6be57d7f1585dd104ac.

**Comment by mrjoro** [comment]
Is this PR still active?

**Comment by sebastianbenz** [comment]
Not anymore.
